src/modules/ens/ens.service.unit.spec.ts

```diff
@@ -122,6 +122,21 @@ describe('EnsService', () => {
     await app.close();
   });
 
+  describe('Sync ENS', () => {
+    it('syncENS() throw an error', async () => {
```

Comment:
It is a bit vague name, because synchronization can throw by many reasons. Would it be more descriptive `deleted namespace synchronization should throw`?